### PR TITLE
feat: add Liquidity Used column to LPPositions

### PR DESCRIPTION
**Pull request by hellodespereaux**
## Scope

This PR adds the column Liquidity Used (%) to LPPositions

## How to Test

Open demo link and deposits some liquidity into CLAMM

### CODE REVIEW

File: apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { useCallback, useMemo } from 'react';
+import { BigNumber } from 'ethers';
```
Comment (witherblock) [inline apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx:2]: We have migrated to viem, do not use ethers

**Review by witherblock — CHANGES_REQUESTED** [review]

File: apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx (hunk 1 above)
Comment (hellodespereaux) [reply apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx]: Sorry, it was a useless import as I then removed BigNumber
Pushed a commit to fix

File: apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx [hunk 2]
```diff
@@ -285,6 +314,14 @@ const LPPositions = ({
               token0Symbol: token0Symbol,
               token1Symbol: token1Symbol,
             },
+            liquidityUsedPercentage: {
+              token0Amount:
+                100 - (100 * token0Withdrawable) / token0LiquidityInToken || 0,
+              token1Amount:
+                100 - (100 * token1Withdrawable) / token1LiquidityInToken || 0,
```
[the remark this answers is not in the document]
Comment (hellodespereaux) [reply apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx:320]: done

File: apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx [hunk 3]
```diff
@@ -128,6 +134,28 @@ const columns = [
...
+        <div className="flex flex-col items-start justify-center">
+          <span>
+            {formatAmount(token0Amount, 2)}
+            {'% '}
+            <span className="text-stieglitz">{token0Symbol}</span>
+          </span>
+          <span>
+            {formatAmount(token1Amount, 2)}
+            {'% '}
+            <span className="text-stieglitz">{token1Symbol}</span>
```
[the remark this answers is not in the document]
Comment (hellodespereaux) [reply apps/dapp/src/components/clamm/PositionsTable/components/Positions/components/LPPositions.tsx:153]: done

**Pull request closed without merging**